docs/workflows/genomic_characterization/theiacov.md

```diff
@@ -323,9 +334,43 @@ The `organism_parameters` sub-workflow is the first step in all TheiaCoV workflo
         | genome_length_input | measles | `16000` |
         | nextclade_dataset_name_input | measles | `"nextstrain/measles/N450/WHO-2012"` |
         | nextclade_dataset_tag_input | measles | `"2025-03-26--11-47-13Z"` |
+        | reference_genome | measles | `"gs://theiagen-public-resources-rp/reference_data/viral/measles/NC_001498.1_measles_reference.fasta"` |
+        | vadr_max_length | measles | `18000` |
+        | vadr_skip_length | measles | `0` |
+        | vadr_mem | measles | `24` |
+        | vadr_options | measles | `"--mkey mev -r --indefclass 0.01"` |
+        | vadr_model_file | measles | `"gs://theiagen-public-resources-rp/reference_data/databases/vadr_models/vadr-models-mev-1.02.tar.gz"` |
 
         </div>
-                
+
+    ??? toggle "Mumps Defaults"
```

Comment:
this is no longer the appropriate format based on a recent update to theiacov. please correct this